homeassistant/components/flux_led/light.py

```diff
@@ -158,7 +157,6 @@ async def async_setup_platform(
                     CONF_MAC: discovered_mac_by_host.get(host),
                     CONF_NAME: device_config[CONF_NAME],
                     CONF_PROTOCOL: device_config.get(CONF_PROTOCOL),
-                    CONF_MODE: device_config.get(ATTR_MODE, MODE_AUTO),
```

Comment:
This is vestigial from when the integration couldn't figure it out automatically.